ship.config.js

```diff
@@ -0,0 +1,30 @@
+/* eslint-disable functional/immutable-data */
+/* eslint-disable import/no-commonjs */
+
+const fs = require('fs');
+const path = require('path');
+
+module.exports = {
+  monorepo: {
+    mainVersionFile: 'package.json',
+    packagesToBump: ['packages/*'],
+    packagesToPublish: ['packages/*'],
+  },
+  getTagName: ({ version }) => `${version}`,
+  conventionalChangelogArgs:
+    '--config conventional-changelog.config.js --infile CHANGELOG.md --same-file',
+  versionUpdated({ version, dir }) {
```

Comment:
You're gonna need to update the version of the the root `package.json` here.

Reply:
This is already done by the `monerepo.mainVersionFile` set to `package.json` on line 9, along with the update of others package-specific `package.json` files.

Here in `versionUpdated`, I'm only manually updating the root `lerna.json` and `packages/client-common/src/version.ts` TypeScript file which aren't done automatically.